src-docs/src/views/tool_tip/tool_tip.js

```diff
@@ -56,6 +56,19 @@ export default () => (
 
     <EuiSpacer />
 
+    <p>
+      <EuiToolTip
+        position="top"
+        content="Here is some tooltip text"
+        display="block">
+        <EuiButton href="#" fullWidth>
```

Comment:
I actually find the redirects frustrating.

```suggestion
        <EuiButton fullWidth>
```